deployment/ccip/changeset/v1_5_1/cs_deploy_ping_pong.go

```diff
@@ -0,0 +1,198 @@
+package v1_5_1
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/ccip/changeset"
+
+	"github.com/smartcontractkit/chainlink-ccip/chains/evm/gobindings/generated/latest/ping_pong_demo"
+)
+
+var _ deployment.ChangeSet[DeployPingPongDemoContractsConfig] = DeployPingPongDemoContractsChangeset
+var _ deployment.ChangeSet[StartPingPongDemoContractsConfig] = StartPingPongDemoContractsChangeset
+var _ deployment.ChangeSet[SetPausedPingPongDemoContractsConfig] = SetPausedPingPongDemoContractsChangeset
+var _ deployment.ChangeSet[SetConterpartPingPongDemoContractsConfig] = SetCounterpartPingPongDemoContractsChangeset
+
+type DeployPingPongDemoContractsConfig struct {
+	ChainSelector uint64
+	IsTestRouter  bool
+}
+
+func (c DeployPingPongDemoContractsConfig) Validate(env deployment.Environment, state changeset.CCIPOnChainState) error {
+	chainState := state.Chains[c.ChainSelector]
+
+	router := chainState.Router
+	if c.IsTestRouter {
+		router = chainState.TestRouter
+	}
+
+	if router == nil {
+		return fmt.Errorf("router address is empty for chain %d", c.ChainSelector)
+	}
+
+	if chainState.LinkToken == nil {
```

Comment:
There are chains in which StaticLinkToken is available instead of LinkToken - you can use this logic - https://github.com/smartcontractkit/chainlink/blob/develop/deployment/ccip/changeset/state.go#L236